fast64_internal/oot/oot_scene_room.py

```diff
@@ -172,13 +172,24 @@ def drawExitProperty(layout, exitProp, index, headerIndex, objName):
 
 class OOTObjectProperty(bpy.types.PropertyGroup):
     expandTab: bpy.props.BoolProperty(name="Expand Tab")
-    objectID: bpy.props.EnumProperty(items=ootEnumObjectID, default="OBJECT_HUMAN")
-    objectIDCustom: bpy.props.StringProperty(default="OBJECT_HUMAN")
+    objectID: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectID, default="obj_human")
+    objectIDLegacy: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectIDLegacy, default="OBJECT_HUMAN")
```

Comment:
this breaks old blends, doesn't it? `objectID` with the new enum won't retain old values with the old enum, they'll be invalid and discarded afaik (but I may be wrong?)